src/main/java/school/lemon/changerequest/java/introduction/hw1/Task2.java

```diff
@@ -3,8 +3,8 @@
 
 public class Task2 {
     public static int getMeters(int centimeters) {
-        double b = .01;//тут мы вносим дабл 0.01
-        return (int) (centimeters * b);//не забываем поставить вывод через int
+
+        return (centimeters/100);//не забываем поставить вывод через int
```

Comment:
parenthesis are unnecessary here.
Also try to avoid such comments, cause right now they are confusing.